### DISCUSSION TITLE
Support for Xinference TTS Model

### DISCUSSION THREAD

**Discussion by liduang** [langgenius/dify#5294] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

Support for Xinference TTS Model

### 2. Describe the feature you'd like to see

Support for Xinference TTS Model

### 3. How will this feature improve your workflow or experience?

Support for Xinference TTS Model

### 5. Can you help us with this feature?

- [ ] I am interested in contributing to this feature.

**Comment by liduang** [comment]
请支持下Xinference的TTS模型类型，谢谢啦

**Comment by halexan** [comment]
Please support the TTS model types for Xinference, thank you.

**Comment by yzning** [comment]
Any plan to support Xinference TTS mode? :-)
